test/auth/oidc_test.go

```diff
@@ -75,7 +75,7 @@ func TestChannelImplSupportsOIDC(t *testing.T) {
 	name := feature.MakeRandomK8sName("channelimpl")
 	env.Prerequisite(ctx, t, channel.ImplGoesReady(name))
 
-	env.Test(ctx, t, oidc.AddressableHasAudiencePopulated(channel_impl.GVR(), channel_impl.GVK().Kind, name, env.Namespace()))
+	env.TestSet(ctx, t, oidc.AddressableOIDCConformance(channel_impl.GVR(), channel_impl.GVK().Kind, name, env.Namespace()))
```

Comment:
Out of curiosity, why are we changing which rekt features are running here? I can't see any commits which include changes to any of these features

Reply:
@Cali0707 As shown here:
https://github.com/knative/eventing/blob/9431ce929a4f368959eb94a62d50de3b7d08b6cc/test/auth/features/oidc/addressable_oidc_conformance.go#L39-L41
`AddressableHasAudiencePopulated` is included